examples_tests/35.TAA/main.cpp

```diff
@@ -0,0 +1,398 @@
+#define _IRR_STATIC_LIB_
+#include <irrlicht.h>
+#include "../ext/FullScreenTriangle/FullScreenTriangle.h"
+
+#define USE_JITTER 1
+
+using namespace irr;
+using namespace core;
+
+bool quit = false;
+//!Same As Last Example
+class MyEventReceiver : public IEventReceiver
+{
+public:
+
+	MyEventReceiver()
+	{
+	}
+
+	bool OnEvent(const SEvent& event)
+	{
+        if (event.EventType == irr::EET_KEY_INPUT_EVENT && !event.KeyInput.PressedDown)
+        {
+            switch (event.KeyInput.Key)
+            {
+            case irr::KEY_KEY_Q: // switch wire frame mode
+                quit = true;
+                return true;
+            default:
+                break;
+            }
+        }
+
+		return false;
+	}
+
+private:
+};
+
+// GLOBAL CONSTANTS
+static const dimension2d<uint32_t> WINDOW_SIZE(1280, 720);
+static const uint32_t WINDOWS_SIZE_array[]{ 1280, 720 };
+static const core::vector2df WINDOW_SIZE_vec2df(1280.f, 720.f);
+constexpr uint32_t JITTER_OFFSET_CNT = 8u;
+static const core::vector2df JITTER_OFFSETS[JITTER_OFFSET_CNT]{
```

Comment:
ok even according to INSIDE and Ke XU
http://advances.realtimerendering.com/s2016/s16_Ke.pptx

That sequence needs to be Halton or Sobol, probably Halton best.